src/main/scala/com/codacy/client/bitbucket/v2/service/IssueServices.scala

```diff
@@ -8,8 +10,9 @@ import play.api.libs.json.{JsString, Json}
 class IssueServices(client: BitbucketClient) {
 
   def createIssue(author: String, repo: String, title: String, body: String): RequestResponse[Issue] = {
-
-    val url = s"https://api.bitbucket.org/2.0/repositories/$author/$repo/issues"
+    val encodedAuthor = URLEncoder.encode(author, "UTF-8")
```

Comment:
Codacy found an issue: [The string literal 'UTF-8' appears 2 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)